File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/Schedule.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class Schedule : DomainResource, IResource
{
    public required Base.ResourceReference[] Actor { get; set; }
    public bool? Active { get; set; }
    public string? Comment { get; set; }
    public Base.CodeableConcept[]? Specialty { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Base.CodeableConcept[]? ServiceType { get; set; }
    public Base.Period? PlanningHorizon { get; set; }
    public Base.CodeableConcept[]? ServiceCategory { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/Schedule.py
from typing import Optional, List
from base import Period
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

class Schedule(DomainResource):
    actor: list[Reference] = []
    active: Optional[bool] = None
    comment: Optional[str] = None
    specialty: Optional[List[CodeableConcept]] = None
    identifier: Optional[List[Identifier]] = None
    service_type: Optional[List[CodeableConcept]] = None
    planning_horizon: Optional[Period] = None
    service_category: Optional[List[CodeableConcept]] = None